### PR TITLE
PnL Fix

### PR DESCRIPTION
**Pull request by DemolaJames**
# Task:
Bug fix for Pnl on partial deposits
## Description

Pnl calculation for crab v2 was changed to (currentPositionValueNow + totalWithdrawnAmount) / totalValueDeposited 

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested

Please describe how to test to verify the changes. Provide instructions so we can reproduce.

## FE Checklist

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] Added video recordings if it is a UI change

## User Facing Checklist

- [ ] I fully understand the user problem this PR is solving
- [ ] I know who the target user is for this PR and have a deep understanding of that user
- [ ] I have tried this flow thinking from the pov of the target user for this PR
- [ ] I (or working w someone on team) have scheduled a user test for this PR (if it is a large change)

### CODE REVIEW

File: packages/frontend/src/hooks/useCrabPosition/useCrabPosition.ts [hunk 1]
```diff
@@ -118,10 +118,10 @@ export const useCrabPositionV2 = (user: string) => {
   const [minPnlUsd, setMinPnlUsd] = useState(BIG_ZERO)
   const [minPnL, setMinPnL] = useState(BIG_ZERO)
 
-  const { depositedEth, usdAmount: depositedUsd } = useAppMemo(() => {
-    if (txHistoryLoading || !txHistoryData) return { depositedEth: BIG_ZERO, usdAmount: BIG_ZERO }
+  const { depositedEth, usdAmount: depositedUsd, withdrawnEth, withdrawnUsdAmount,depositedValueEth,depositedValueUsd } = useAppMemo(() => {
```
Comment (sanandnarayan) [inline packages/frontend/src/hooks/useCrabPosition/useCrabPosition.ts:121]: @KMKoushik can you confirm that adding these consts would not break any other part of the code?

**Comment by sanandnarayan** [comment]
LGTM . @KMKoushik can you review this once.

**Review by sanandnarayan — APPROVED** [review]

File: packages/frontend/src/hooks/useCrabPosition/useCrabPosition.ts (hunk 1 above)
Comment (KMKoushik) [reply packages/frontend/src/hooks/useCrabPosition/useCrabPosition.ts:121]: @DemolaJames Let's sync on this. I didn't understand the issue or fix completely!

**Comment by DemolaJames** [comment]
branched out to...https://github.com/opynfinance/squeeth-monorepo/pull/644
Pr merged

**Pull request closed without merging**